cloverage/test/cloverage/instrument_test.clj

```diff
@@ -217,10 +241,32 @@
           (let [[[message-type _ message]] @log-messages]
             (t/is (= :error
                      message-type))
-            (doseq [s ["Error instrumenting form"
+            (doseq [s ["Error evaluating instrumented form"
```

Comment:
The reworked error-handling code now differentiates between an error that happens during instrumentation and an error that happens while evaluating the instrumented form, so I had to update this test.